Readme.md

```diff
@@ -61,22 +61,20 @@ play along! :tada:
 1. Clone this repo.
 2. Get [Node/NPM](https://nodejs.org) and [yarn](https://yarnpkg.com)
    (`npm install -g yarn`).
-3. Run `make` to install dependencies and compile the source files.
+3. Run `yarn` to install dependencies and compile the source files.
 4. Load it in Firefox or Chromium/Chrome:
-    * In Firefox (≥49): run `npm run firefox` (or run [web-ext] directly for more control).
+    * In Firefox (≥49): run `yarn run firefox` (or run [web-ext] directly for more control).
```

Comment:
`yarn run $SCRIPT_NAME` is equivalent to `yarn $SCRIPT_NAME`, assuming `$SCRIPT_NAME` is not a reserved `yarn` cmd (`add`, `remove`, etc.). Actually doesn't really matter - either should be fine here - just me being nitpicky 😄